services/spar/src/Spar/Scim/User.hs

```diff
@@ -33,7 +33,8 @@ import Data.Range
 import Data.String.Conversions
 import Galley.Types.Teams    as Galley
 import Network.URI
-import Spar.App (Spar, Env, wrapMonadClient, sparCtxOpts, createUser_, wrapMonadClient)
+
```

Comment:
why the empty line?  (not important.)

Reply:
Separates the 'local' spar imports from the external stuff; must be my old PEP8 python style leaking thru 😄 

The rest of the imports are inconsistent anyway haha.

Reply:
> Separates the 'local' spar imports from the external stuff

I usually do that as well